flow/connectors/postgres/qrep_query_executor_test.go

```diff
@@ -214,28 +181,20 @@ func TestAllDataTypes(t *testing.T) {
 	}
 
 	expectedText := "text"
-	if record[5].Value().(string) != expectedText {
-		t.Fatalf("expected %v, got %v", expectedText, record[5].Value())
-	}
+	require.Equal(t, expectedText, record[5].Value(), "expected 'text'")
 
 	expectedBytea := []byte("bytea")
-	if !bytes.Equal(record[6].Value().([]byte), expectedBytea) {
-		t.Fatalf("expected %v, got %v", expectedBytea, record[6].Value())
-	}
+	require.Equal(t, expectedBytea, record[6].Value(), "expected 'bytea'")
 
 	expectedJSON := `{"key":"value"}`
-	if record[7].Value().(string) != expectedJSON {
-		t.Fatalf("expected %v, got %v", expectedJSON, record[7].Value())
-	}
+	require.Equal(t, expectedJSON, record[7].Value(), "expected '{\"key\":\"value\"}'")
 
 	actualUUID := record[8].Value().([16]uint8)
-	if !bytes.Equal(actualUUID[:], savedUUID[:]) {
-		t.Fatalf("expected %v, got %v", savedUUID, actualUUID)
-	}
+	require.Equal(t, savedUUID[:], actualUUID[:], "expected savedUUID: %v", savedUUID)
+	// not sure why the values are unequal
```

Comment:
might be caused by PG being microsecond precision. Try rounding `time.Now()` to seconds